media-sound/pulseaudio-modules-bt/pulseaudio-modules-bt-1.3-r1.ebuild

```diff
@@ -53,6 +53,16 @@ load-module module-bluetooth-discover
 .endif
 "
 
+src_configure() {
+	local mycmakeargs=(
+	-DCODEC_AAC_FDK="$(usex fdk)"
```

Comment:
I think this needs an indentation bump.

Reply:
Will apply. I just followed another ebuild at portage using that format.

My next git push will contain it.

Thanks for reviewing

Reply:
No problem, looking good to me 👍